src/rest/Facet/FacetRequest.ts

```diff
@@ -41,6 +45,22 @@ export interface IFacetRequestValue extends IRangeValue {
    * **Default:** `false`
    */
   preventAutoSelect?: boolean;
+  /**
+   * Whether to retrieve the children of this hierarchical facet value. Can only be used on leaf values.
```

Comment:
@fbeaudoincoveo: do you think it would be better to use `...categorical facet value...` instead of `hierarchical` to avoid mixing terminology with the deprecated component?

Same comment for the other instances on line 55, 59, 60.

Reply:
Yeah I'll change it

Reply:
I would change it for `(i.e., values with no current children)`